packages/audit/hooks/event_logger.py

```
@@ -12,10 +12,50 @@
 import re
+import subprocess
 import tempfile
 import uuid
 from typing import Any
 
+
+def _resolve_root_worktree() -> str | None:
+    """Git の root worktree パスを解決する。
+
+    worktree 環境では main worktree のパスを返す。
+    通常リポジトリではリポジトリルートを返す（動作変更なし）。
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)